Modules/QuestieEventHandler.lua

```diff
@@ -60,21 +62,19 @@ local QuestieFramePool = QuestieLoader:ImportModule("QuestieFramePool")
 local QuestieOptionsDefaults = QuestieLoader:ImportModule("QuestieOptionsDefaults")
 ---@type ZoneDB
 local ZoneDB = QuestieLoader:ImportModule("ZoneDB")
----@type HBDHooks
-local HBDHooks = QuestieLoader:ImportModule("HBDHooks")
 
 --- LOCAL ---
 --False -> true -> nil
 local didPlayerEnterWorld = false
-local hasFirstQLU = false
 local shouldRunQLU = false
 
 local LibDropDown = LibStub:GetLibrary("LibUIDropDownMenu-4.0")
 
 local continueInit
 
+function QuestieEventHandler:RegisterEarlyEvents(callback)
 
-function QuestieEventHandler:RegisterAllEvents(callback)
+    continueInit = callback
```

Comment:
Can we still rename  `continueInit` and extract it into a proper function that is created outside of this method?